website/docs/intro.md

```diff
@@ -2,7 +2,18 @@
 sidebar_position: 1
 ---
 
-# Introduction
-
-> Work in progress...
+# General
 
+> About plugin
+What is ECInventory for? It is a powerful tool for changing your inventory to your liking. With it, you can make your server even more atmospheric. You will be able to add a lot of new highly customizable inventories. Thanks to the Mimic API, ECInventory can use the capabilities of other plugins (MMOItems, MMOCore, QuantumRPG, SkillAPI, PROSkillAPI, Heroes, etc.).
+---
+# Where to begin? And what functionality does ECInventory contribute?
+> - Allows create more than one inventory
```

Comment:
It's better to separate heading from paragraph with spaces to make it visually cleaner:
```suggestion
---

# Where to begin? And what functionality does ECInventory contribute?

> - Allows create more than one inventory
```

Reply:
Also, I think heading should be level 2 instead of 1, and better to make it shorter:
```md
## Features
```